aip/general/0145.md

```diff
@@ -48,6 +48,25 @@ Exclusive end values are preferable for the following reasons:
 - It is easier to reason about abutting ranges: `[0, x), [x, y), [y, z)`, where
   values are chainable from one range to the next.
 
+### Timestamp intervals
+
+The following section describes the use of the [google.type.Interval][interval]
+type, found amongst the common protos that are described in [AIP-213][].
+This type represents a range between two timestamps, with an inclusive start
+value and exclusive end value as described above.
```

Comment:
I think we can remove, "as described above". I apologize if I was the one that requested that.

Otherwise, LGTM.

Reply:
Done, thanks.